Bai1/main.js

```diff
@@ -0,0 +1,95 @@
+function submit() {
+    var name = document.getElementById('name').value;
+    var password = document.getElementById('password').value;
+    var email = document.getElementById('email').value;
+    var job = document.getElementById('job').value;
+    var accept = document.getElementById('accept').checked;
+    var male = document.getElementById('male');
+    var female = document.getElementById('female');
+    var sex;
+    if (!male.checked && !female.checked) {
+        sex = "Female";
+    } else {
+        if (!male.checked) {
+            sex = "Female";
+        } else {
+            sex = "Male";
+        }
+    }
+    checkName(name);
+    checkPassword(password);
+    checkEmail(email);
+    checkJob(job);
+    checkAccept(accept);
+    if (checkName(name) && checkPassword(password) && checkEmail(email) && checkJob(job) && checkAccept(accept)) {
+        alert(
+            "name: " + name + ", " +
+            "\npassword: " + password + ", " +
+            "\nemail: " + email + ", " +
+            "\nsex: " + sex + ", " +
+            "\njob: " + job
+        );
+    }
+}
+
+function checkName(name) {
+    if (name === '') {
+        document.getElementById('name_error').innerHTML = "Name must be filled out";
+        return false;
+    } else {
+        document.getElementById('name_error').innerHTML = "";
+        return true;
+    }
```

Comment:
Không cần thiết else, có lỗi sẽ vào if không có lỗi thì sẽ return thôi.
Tương tự fix cho những function khác
```suggestion
    document.getElementById('name_error').innerHTML = "";
    return true;
```